src/slack/slack.js

```diff
@@ -1159,9 +1159,15 @@ export class Slack {
 
   async getOrderAndItemsFromDb(orderId) {
     return await knex.transaction(async (trx) => {
+      const fields = ['id', 'isCompany', 'user', 'office', 'isUrgent', 'isHome']
+
+      if (this.variant !== 'wincent') {
+        fields.push('paymentType')
+      }
+
       const order = (
         await trx
-          .select('id', 'isCompany', 'user', 'office', 'isUrgent', 'isHome', 'paymentType')
```

Comment:
I have code-searched for other instances of paymentType query and did not see other cases